engines/experiment/pkg/request/request_test.go

```diff
@@ -44,6 +44,17 @@ func TestGetValueFromHTTPRequest(t *testing.T) {
 			}(),
 			expected: "123",
 		},
+		"success | header multiple value": {
+			field:    "CustomerID",
+			fieldSrc: request.HeaderFieldSource,
+			header: func() http.Header {
+				header := http.Header{}
+				header.Set("CustomerID", "123")
+				header.Add("CustomerID", "321")
+				return header
+			}(),
+			expected: "123,321",
+		},
```

Comment:
Just curious, how would the header look like if its corresponding value is an object? :o

Reply:
it depends on the library and how you create the headers. It will just be a string nevertheless. Eg ```header.add(`{"a":"b"}``` will result in ```123,{"a","b"}```